plugins/storage/rdbms/init.go

```diff
@@ -124,6 +124,16 @@ func (r *RDBMS) Version() (uint64, error) {
 	return migrationlib.DBVersion(r.db)
 }
 
+// Reset wipes entire database contents. Used in tests.
+func (r *RDBMS) Reset() error {
```

Comment:
Scary stuff. Should we at least yell a bit, like https://github.com/facebookincubator/contest/blob/master/plugins/targetlocker/dblocker/dblocker.go#L343 ?

Also DELETE vs. TRUNCATE. (not sure, just wanted to mention it)

Reply:
there is no context to yell to though, the storage api doesn't use context (yet).

Reply:
changed to TRUNCATE